.github/workflows/auto-approve-bundled-package-updates.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  pull_request_target:
```

Comment:
this is only running off branches in the elastic/kibana repository?  we can swap it with `pull_request`